imgui/core.pyx

```diff
@@ -2638,7 +2723,241 @@ def is_item_hovered():
+
+    Returns:
+        bool: True if any item is active, otherwise False.
+
+    .. wraps::
+        bool IsAnyItemActive()
+    """
+    return cimgui.IsAnyItemActive()
+
+
+def get_item_rect_min():
+    """Get bounding rect of the last item in screen space.
+
+    Returns:
+        Vec2: item minimum boundaries two-tuple ``(width, height)``
+
+    .. wraps::
+        ImVec2 GetItemRectMin()
+    """
+    return _cast_ImVec2_tuple(cimgui.GetItemRectMin())
+
+
+def get_item_rect_max():
+    """Get bounding rect of the last item in screen space.
+
+    Returns:
+        Vec2: item maximum boundaries two-tuple ``(width, height)``
+
+    .. wraps::
+        ImVec2 GetItemRectMax()
+    """
+    return _cast_ImVec2_tuple(cimgui.GetItemRectMax())
+
+
+def get_item_rect_size():
+    """Get bounding rect of the last item in screen space.
+
+    Returns:
+        Vec2: item boundaries two-tuple ``(width, height)``
+
+    .. wraps::
+        ImVec2 GetItemRectSize()
+    """
+    return _cast_ImVec2_tuple(cimgui.GetItemRectSize())
+
+
+def set_item_allow_overlap():
+    """Allow last item to be overlapped by a subsequent item.
+    Sometimes useful with invisible buttons, selectables, etc.
+    to catch unused area.
+
+    .. wraps::
+        void SetItemAllowOverlap()
+    """
+    cimgui.SetItemAllowOverlap()
+
+
+def is_window_hovered():
+    """Is current window hovered and hoverable (not blocked by a popup)
+    Differentiate child windows from each others.
+
+    Returns:
+        bool: True if current window is hovered, otherwise False.
+
+    .. wraps::
+        bool IsWindowHovered()
+    """
+    return cimgui.IsWindowHovered()
+
+
+def is_window_focused():
+    """Is current window focused.
+
+    Returns:
+        bool: True if current window is on focus, otherwise False.
+
+    .. wraps::
+        bool IsWindowFocused()
+    """
+    return cimgui.IsWindowFocused()
+
+
+def is_root_window_focused():
+    """Is root window focused.
+
+    Returns:
+        bool: True if root window is on focus, otherwise False.
+
+    .. wraps::
+        bool IsRootWindowFocused()
+    """
+    return cimgui.IsRootWindowFocused()
+
+
+def is_root_window_or_any_child_focused():
+    """Is current root window or any of its child
+    (including current window) focused.
+
+    Returns:
+        bool: True if any of the windows is on focus, otherwise False.
+
+    .. wraps::
+        bool IsRootWindowOrAnyChildFocused()
+    """
+    return cimgui.IsRootWindowOrAnyChildFocused()
+
+
+def is_root_window_or_any_child_hovered():
+    """Is current root window or any of its child
+    (including current window) hovered.
+
+    Returns:
+        bool: True if any of the windows is hovered, otherwise False.
+
+    .. wraps::
+        bool IsRootWindowOrAnyChildHovered()
+    """
+    return cimgui.IsRootWindowOrAnyChildHovered()
+
+
+def is_rect_visible(float size_width, float size_height):
+    """Test if rectangle of given size starting from cursor pos
```

Comment:
Could you reformat this docstring to be more consistent with existing documentation style?